README.md

```diff
@@ -15,6 +15,23 @@ ember install ember-frost-info-bar
 ```
 
 ## Examples
+### Block Format
+```handlebars
+{{#frost-info-bar as |slot|}}
+  {{#block-slot slot 'title'}}
+      <div class='primary-title'>User accounts</div>
```

Comment:
The div should move into the component template and sub-title === summary.  So this format should be 

```handlebars
{{#frost-info-bar as |slot|}}
  {{#block-slot slot 'title'}}
    User accounts
  {{/block-slot}}
  {{#block-slot slot 'summary'}}
    Summary stats
  {{/block-slot}}
  {{#block-slot slot 'actions' as |action|}}
    {{action.button 
      icon='frost/infobar-create' 
      text='Create' 
      onClick=(action 'createUser')
    }}
  {{/block-slot}}
{{/frost-info-bar}}